src/Console/ConsoleOptionParser.php

```diff
@@ -173,7 +173,7 @@ public function __construct($command = null, $defaultOptions = true)
      */
     public static function create($command, $defaultOptions = true)
     {
-        return new ConsoleOptionParser($command, $defaultOptions);
+        return new static($command, $defaultOptions);
```

Comment:
I think this is fine. It should break backwards compatibility unless there is an incompatible subclass which is not our purview.